tests/spec/puppeteer-axe-spec.js

```diff
@@ -0,0 +1,80 @@
+/* Puppeteer tests for the A11y. */
+const { AxePuppeteer } = require('axe-puppeteer');
+const puppeteer = require('puppeteer');
+
+describe("Puppeteer accessibility tests for the Python Editor.", function() {
+    'use strict';
+
+    const editorURL = 'http://localhost:5000/editor.html';
+    const helpURL = 'http://localhost:5000/help.html';
+    let browser = null;
+
+    beforeAll(async () => {
+        // Setup a headless Chromium browser.
+        // Flags allow Puppeteer to run within a container.
+        browser = await puppeteer.launch({
+            headless: true,
+            args: ["--no-sandbox", "--disable-setuid-sandbox", "--disable-dev-shm-usage"]
+        });
+    });
+
+    afterAll(async () => {
+        browser.close();
+    });
+
+    test( 'Checks the editor.html page with Axe', async () => {
+        // First, run some code which loads the content of the page.
+        const page = await browser.newPage();
+        await page.goto(editorURL);
+        await new AxePuppeteer(page).analyze();
+        await expect(page).toPassAxeTests({
+            // disable tab-index as we have values greater than 0
+            // and h1 as we aren't using this heading
```

Comment:
Glad to see a comments here and the other tests cases explaining why these rules are ignored, good call 👍